defaults/main.yml

```diff
@@ -201,3 +201,22 @@ superset_dashboard_position_data_limit: 65535
 superset_virtualenv_path: "{{ superset_home }}/.virtualenvs/{{ superset_user }}"
 superset_python_path: "{{ superset_virtualenv_path }}/lib/{{ superset_python_executable }}"
 superset_base_path: "{{ superset_python_path }}/site-packages/superset"
+
+# uwsgi configs
+superset_wsgi_socket: "{{ superset_codebase_path }}/superset.sock"
+superset_wsgi_chmod_socket: "664"
+superset_codebase_path: "/tmp"
```

Comment:
Would it make sense if the default codebase path was `"{{ superset_home }}/app"`?

Reply:
Yeah, okay

Reply:
Cool